src/Appwrite/Utopia/Response/Model/Func.php

```diff
@@ -87,6 +99,54 @@ public function __construct()
                 'default' => 15,
                 'example' => 1592981237,
             ])
+            ->addRule('entrypoint', [
+                'type' => self::TYPE_STRING,
+                'description' => 'The entrypoint file used to execute the deployment.',
+                'default' => '',
+                'example' => 'index.js',
+            ])
+            ->addRule('buildCommand', [
+                'type' => self::TYPE_STRING,
+                'description' => 'The build command used to build the deployment.',
+                'default' => '',
+                'example' => 'npm run build',
+            ])
+            ->addRule('installCommand', [
+                'type' => self::TYPE_STRING,
+                'description' => 'The install command used to build the deployment.',
+                'default' => '',
+                'example' => 'npm install',
+            ])
+            ->addRule('vcsInstallationId', [
+                'type' => self::TYPE_STRING,
+                'description' => 'Function vcs installation id.',
+                'default' => '',
+                'example' => '644051bd6572792165cc',
+            ])
+            ->addRule('vcsRepositoryId', [
+                'type' => self::TYPE_STRING,
+                'description' => 'Git Repository ID',
+                'default' => '',
+                'example' => 'appwrite',
+            ])
+            ->addRule('vcsBranch', [
+                'type' => self::TYPE_STRING,
+                'description' => 'Git branch name',
```

Comment:
Do you want me to make sure to use VCS instead of Git in all response models?
Buut saying `VCS (Version Control System) ` every single time instead of `git` feels terrible.